src/skmultiflow/evaluation/base_evaluator.py

```diff
@@ -607,80 +678,137 @@ def evaluation_summary(self):
+                            data_id=constants.MEAN)[i]))
             if constants.AMAE in self.metrics:
-                print('{} - AMAE          : {:4f}'.format(
-                    self.model_names[i],
-                    self._data_buffer.get_data(metric_id=constants.AMAE, data_id=constants.MEAN)[i]))
+                print(
+                    '{} - AMAE          : {:4f}'.format(
+                        self.model_names[i],
+                        self._data_buffer.get_data(
+                            metric_id=constants.AMAE,
+                            data_id=constants.MEAN)[i]))
             if constants.ARMSE in self.metrics:
-                print('{} - ARMSE          : {:4f}'.format(
-                    self.model_names[i],
-                    self._data_buffer.get_data(metric_id=constants.ARMSE, data_id=constants.MEAN)[i]))
+                print(
+                    '{} - ARMSE          : {:4f}'.format(
+                        self.model_names[i],
+                        self._data_buffer.get_data(
+                            metric_id=constants.ARMSE,
+                            data_id=constants.MEAN)[i]))
             if constants.RUNNING_TIME in self.metrics:
                 # Running time
-                print('{} - Training time (s)  : {:.2f}'.format(
-                    self.model_names[i], self._data_buffer.get_data(metric_id=constants.RUNNING_TIME,
-                                                                    data_id='training_time')[i]))
-                print('{} - Testing time  (s)  : {:.2f}'.format(
-                    self.model_names[i], self._data_buffer.get_data(metric_id=constants.RUNNING_TIME,
-                                                                    data_id='testing_time')[i]))
-                print('{} - Total time    (s)  : {:.2f}'.format(
-                    self.model_names[i], self._data_buffer.get_data(metric_id=constants.RUNNING_TIME,
-                                                                    data_id='total_running_time')[i]))
+                print(
+                    '{} - Training time (s)  : {:.2f}'.format(
+                        self.model_names[i],
+                        self._data_buffer.get_data(
+                            metric_id=constants.RUNNING_TIME,
+                            data_id='training_time')[i]))
+                print(
+                    '{} - Testing time  (s)  : {:.2f}'.format(
+                        self.model_names[i],
+                        self._data_buffer.get_data(
+                            metric_id=constants.RUNNING_TIME,
+                            data_id='testing_time')[i]))
+                print(
+                    '{} - Total time    (s)  : {:.2f}'.format(
+                        self.model_names[i],
+                        self._data_buffer.get_data(
+                            metric_id=constants.RUNNING_TIME,
+                            data_id='total_running_time')[i]))
```

Comment:
Although technically consistent with PEP8, these print commands on which the content only start in the next line look odd to me. I suggest formatting them in a different way. Perhaps keeping the characters till `.format(` in the first line would be a solution.

Reply:
I agree! Let me correct it!